net/samba4/Makefile

```diff
@@ -303,15 +369,16 @@ define Build/Configure
 	echo 'Checking uname version type: "$(VERSION_DIST) Linux-$(LINUX_VERSION) $(shell date +%Y-%m-%d)"' >> $(PKG_BUILD_DIR)/cross-answers.txt
 	# NOTE: For some unknown reason this answer is not needed on some hosts/distros, yet needed on others?
 	echo 'Checking whether POSIX capabilities are available: OK' >> $(PKG_BUILD_DIR)/cross-answers.txt
-	$(call Build/Configure/Default,configure)
+	$(call Build/Configure/Default)
 endef
 
 # BUG: We need to use "waf install --targets=" otherwise a "make install" or "waf install" will retrigger a full recompile of all possible targets!
 define Build/Compile
 	(cd $(PKG_BUILD_DIR); \
-		./buildtools/bin/waf install -j$(shell nproc) \
-		--targets=$(SAMBA4_MODULES)$(SAMBA4_MODULES_SHARDED)$(BUILD_TARGETS_SERVER),$(BUILD_TARGETS_UTILS),$(BUILD_TARGETS_ADMIN),$(BUILD_TARGETS_CLIENT) \
+		./buildtools/bin/waf install \
+		--jobs=$(shell nproc) \
```

Comment:
This should use to the value set by toolchain, not blindly override it?

Reply:
If you can tell me how to get this value? I could not find any variable that stores this.

Reply:
https://github.com/openwrt/openwrt/blob/master/tools/cmake/Makefile#L23
A bit short on time right now but it looks like cmake uses this?

Reply:
ok thanks, found it:
PKG_JOBS?=$(if $(PKG_BUILD_PARALLEL),$(MAKE_J),-j1)

Reply:
Yeah i remember, i tried this way back and this is not working. 
What i get is: `waf install --jobserver-auth=3,4 ` instead of `waf install -j4` via `waf install $(PKG_JOBS)`
`make -j8` gives me `waf: error: no such option: --jobserver-auth`

So i don't see a easy way to get the make jobs?

Reply:
Yeah, doesn't like there's a clean way of doing it :-(